test/cri-containerd/runpodsandbox_test.go

```diff
@@ -1237,6 +1237,115 @@ func Test_RunPodSandbox_MultipleContainersSameVhd_WCOW(t *testing.T) {
 	}
 }
 
+func Test_RunPodSandbox_CoreDump_LCOW(t *testing.T) {
+	requireFeatures(t, featureLCOW)
+
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause, imageLcowAlpine})
+
+	client := newTestRuntimeClient(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// core is the actual filename for the coredump to get generated and not just the directory that dumps
+	// it will get placed in. So `ls /coredumps/core` will be checking the existence of the file.
+	sbRequest := getRunPodSandboxRequest(t, lcowRuntimeHandler, map[string]string{
+		oci.AnnotationContainerProcessDumpLocation: "/coredumps/core",
+	})
+
+	podID := runPodSandbox(t, client, ctx, sbRequest)
+	defer removePodSandbox(t, client, ctx, podID)
+	defer stopPodSandbox(t, client, ctx, podID)
+
+	mounts := []*runtime.Mount{
+		{
+			HostPath:      "sandbox:///coredump",
+			ContainerPath: "/coredumps",
```

Comment:
is there a situation where we would want multiple containers to share a dump location or be able to see each other's dumps?

Reply:
That's the current scenario. The vision was all containers in a pod would have their core dumps sent to a sandbox mount and a sidecar would actually do something with them.